### ISSUE TITLE
Validation set & test/train answer differences

### ISSUE THREAD

**Issue by maximek3** [sarahESL/PubMedCLIP#4]
Hello,

First off, thanks for this contribution and for making the code public. 

I have some questions regarding the validation of your model on VQA-Rad:

First, I wasn’t able to find anything about a validation set. Is the current setup that the model is validated and tested on the same test set?

Second, as I understand it the problem is set up as classification of all the possible answers present in the dataset. However, I noticed that there are many answers in the test set that are not present in the train set. Wouldn’t this mean that it’s impossible (as long as we don’t include any textual embedding of the answers) for the model to predict these answers correctly?

Thanks!

**Comment by sarahESL** [comment]
Thanks for your inquiry.

1. Yes, the test set has been used as validation set as well.

2. This is mainly true for the VQA-RAD dataset. In the VQA-RAD dataset, (if I remember correctly) about 10% of the answers present in the test set are not present in the training set if you exact match them! This potentially means that without zero-shot approaches, we cannot achieve better than 90% overall accuracy. However, for the SLAKE dataset, there are very few answers in test set that are unseen in the training set.
